lib/sycamore/sycamore/query/execution/sycamore_operator.py

```diff
@@ -143,9 +143,11 @@ def execute(self) -> Any:
         os_query = get_knn_query(query_phrase=self.logical_node.query_phrase, context=self.context)
         if self.logical_node.opensearch_filter:
             os_query["query"]["knn"]["embedding"]["filter"] = self.logical_node.opensearch_filter
-        result = self.context.read.opensearch(
-            index_name=self.logical_node.index, query=os_query, reconstruct_document=True
-        ).rerank(query=self.logical_node.query_phrase)
+# XXX XXX XXX MDW 21 Oct 2024 - Disable .rerank() for the Luna demo since it seems to hurt performance a lot.
```

Comment:
This is one we can't just upstream in this case. I don't know quite what to do here. It probably needs to be an option on the QueryVectorDatabase operator. For now let's just leave this hack on the branch.